pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute  npm packages binaries.
+
+- Execute the npm packages binaries:
+
+`npx {{module_name}}`
+
+- Define the package to be installed, then execute the binaries:
```

Comment:
```
Specify a given package name for an executable, in case the binary name does not match the package name
```

Reply:
sorry, you means  I can give a example to replace the module_name?

Reply:
The `-p` flag is necessary when a package has multiple binaries. Otherwise, the package name has to be the same as the binary name. Therefore, my description clarifies in which case you need a package_name and a module_name. Does that make sense ?

Reply:
sorry, I can't fully understand what's your saying, but I will try my best to description, wait monent.